python/ganga/__init__.py

```diff
@@ -20,8 +20,10 @@ def ganga_license():
 from Ganga.Core.InternalServices import ShutdownManager
 ShutdownManager.install()
 
-## TODO need to implement loading of the config system properly here.
-## loadPlugins and autoPopulateGPI will take this into account when loading objects
+system_vars = {}
+for opt in Ganga.Utility.Config.getConfig('System'):
+    system_vars[opt] = Ganga.Utility.Config.getConfig('System')[opt]
```

Comment:
Can we just rename this ```configure``` function to reflect what it's doing in this case? If it's not obvious what it does then rename it to ```setupConfigurationSystem``` or ```applyConfigurationToSession``` or ```addConfigurationFiles``` or something reflecting more what it does (and I don't know what it does so probably all 3 of these names are bad).
Just having the full path to it in the code doesn't help understand what it's doing imo.